.github/PROCESS.md

```diff
@@ -165,3 +207,23 @@ Hotfixes bypass `master` and should only be used for urgent fixes that can't wai
 1. Click **Merge pull request**. Use the dropdown to select this option if necessary.
 
     <img width="191" alt="Merge pull request button" src="https://user-images.githubusercontent.com/236501/47032669-8be1b980-d138-11e8-9a90-d1518c223184.png">
+
+
+## Releasing
+
+1. Create the release branch from `master`, for example: `release-4.1.0`.
+1. Submit a pull request to merge the release branch into `stable`.
+1. Verify all tests are passing, fix any bugs if needed and make sure no undesired commits are in.
+1. Navigate to the root of the repository.
+1. Run `npm i` if it hasn't already been done.
+1. Run `npm run release.prepare`
+    - Select the version based on the type of commits and the [Ionic Versioning](https://ionicframework.com/docs/intro/versioning)
+    - After the process completes, verify the version number in all packages (`core`, `docs`, `angular`)
+    - Verify the changelog commits are accurate and follow the [proper format]((https://github.com/ionic-team/ionic/blob/master/.github/CONTRIBUTING.md#commit-message-format))
+    - Commit these changes with the version number as the message, e.g. `git commit -m "4.1.0"`
+1. Run `npm run release`
+1. Click **Merge pull request**. Use the dropdown to select this option if necessary. This will preserve the commit history from `master` by creating a merge commit.
+
+    <img width="191" alt="Merge pull request button" src="https://user-images.githubusercontent.com/236501/47032669-8be1b980-d138-11e8-9a90-d1518c223184.png">
+
+1. Merge `stable` back into `master` to ensure any changes made on the release branch get added to future releases.
```

Comment:
How?

Reply:
I think this should say:

Merge the release branch back into `master` to ensure any changes made on the release branch get added to future releases.